demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -0,0 +1,891 @@
+                    valid_directory.add(self.run_validations_on_file(file_path, pack_error_ignore_list))
+
+            else:
+                valid_directory.add(self.run_validation_on_package(file_path, pack_error_ignore_list))
+
+        return all(valid_directory)
+
+    def run_validation_on_package(self, package_path, pack_error_ignore_list):
+        valid_package = set()
+
+        for file_name in os.listdir(package_path):
+            file_path = os.path.join(package_path, file_name)
+            if file_path.endswith('.yml') or file_path.endswith('.md'):
+                valid_package.add(self.run_validations_on_file(file_path, pack_error_ignore_list))
+
+        return all(valid_package)
+
+    # flake8: noqa: C901
+    def run_validations_on_file(self, file_path, pack_error_ignore_list, is_modified=False,
+                                old_file_path=None, modified_files=None, added_files=None):
+        """Choose a validator to run for a single file. (i)
+
+        Args:
+            modified_files: A set of modified files - used for RN validation
+            added_files: A set of added files - used for RN validation
+            old_file_path: The old file path for renamed files
+            pack_error_ignore_list: A dictionary of all pack ignored errors
+            file_path: the file on which to run.
+            is_modified: whether the file is modified or added.
+
+        Returns:
+            bool. true if file is valid, false otherwise.
+        """
+        file_type = find_type(file_path)
+
+        if file_type in self.skipped_file_types:
+            self.ignored_files.add(file_path)
+            return True
+
+        elif file_type is None:
+            error_message, error_code = Errors.file_type_not_supported()
+            if self.handle_error(error_message=error_message, error_code=error_code, file_path=f'\n{file_path}'):
+                return False
+
+        if not self.check_only_schema:
+            click.echo(f"\nValidating {file_path} as {file_type.value}")
+
+        structure_validator = StructureValidator(file_path, predefined_scheme=file_type,
+                                                 ignored_errors=pack_error_ignore_list,
+                                                 print_as_warnings=self.print_ignored_errors, tag=self.prev_ver,
+                                                 old_file_path=old_file_path)
+
+        click.secho(f'Validating scheme for {file_path}')
+        if not structure_validator.is_valid_file():
+            return False
+
+        elif self.check_only_schema:
+            return True
+
+        if self.validate_in_id_set:
+            click.echo(f"Validating id set registration for {file_path}")
+            if not self.id_set_validator.is_file_valid_in_set(file_path):
+                return False
+
+        # No validators for reports nor connections
+        if file_type in {FileType.REPORT, FileType.CONNECTION}:
+            return True
```

Comment:
let's add this to the skipped filese

Reply:
added a clarifying comment instead - these files are not ignored they just don't have additional vlidators